css/style.css

```diff
@@ -1,9 +1,243 @@
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* font-family: 'Montserrat', sans-serif;
+font-family: 'M PLUS 1p', sans-serif;
+font-family: 'Work Sans', sans-serif; */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  outline: 0;
+  border: none;
+}
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  text-align: center;
+}
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+header,
+main,
+footer {
+  max-width: 1100px;
+  margin: 0 auto;
+}
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+header {
+  height: 4.3rem;
+  align-items: center;
+  top: 0px;
+  background-color: rgb(248, 248, 248);
+}
+
+.header {
+  height: 4rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  /* background-color: teal;
+  position: fixed; */
+  z-index: 1;
+  margin: 0 7rem;
+  background-color: rgb(248, 248, 248);
+}
+
+.logo-image {
+  height: 40px;
+  width: auto;
+}
+
+.nav label,
+#hamburger {
+  display: none;
+}
+
+.nav-menu {
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  font-size: 1rem;
+  font-weight: 400;
+}
+
+.list-item {
+  margin-left: 25px;
+  padding: 0.6rem 0;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: rgb(148, 143, 143);
+}
+
+.nav-link:hover {
+  color: #e0633a;
+}
+
+.nav-active {
+  color: rgb(75, 73, 73);
+  font-weight: 500;
+}
+
+main {
+  /* margin-top: 5rem; */
+}
+
+.home {
+  background-image: url("../img/first-background.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  height: 35rem;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+}
+
+.content {
+  min-width: 35rem;
+  margin: auto;
+  color: azure;
+  padding-bottom: 3.5rem;
+}
+
+.content h2 {
+  font-weight: 300;
+  font-size: 2.7rem;
+  margin-bottom: 0;
+}
+.content p {
+  font-size: 1.6rem;
+  font-weight: 300;
+  margin-top: 1.2rem;
+}
+.content .learn-more {
+  color: azure;
+  background-color: #e0633a;
+  border: none;
+  border-radius: 4px;
+  padding: 8px 10px;
+  margin-top: 2.8rem;
+  font-size: 0.9rem;
+  font-family: sans-serif;
+  font-weight: 500;
+}
+
+.content .learn-more:hover {
+  color: #e0633a;
+  background-color: azure;
+}
+
+.products {
+  margin: 4rem 0;
+  text-align: center;
+}
+
+.product-cards {
+  display: flex;
+  justify-content: space-around;
+  margin: 0 5.5rem;
+}
+
+.products h2 {
+  font-family: "Montserrat", sans-serif;
+  font-size: 2.2rem;
+  padding: 1rem;
+}
+
+.products p {
+  font-size: 1.2rem;
+  padding: 1.1rem 0;
+}
+
+.cards {
+  padding: 1.3rem 0;
+}
+
+.card-logos {
```

Comment:
All really good class names here - makes it instantly understandable which part of the page you're styling ⭐